docs/bookstore/page-0/page-0-intro.md

```diff
@@ -0,0 +1,159 @@
+
+## **What are we building?**
+
+![App Diagram](images/2.png)
+
+Our App is an online bookstore that sells a single book. Customers can post comments about the book anonymously, with each comment displayed alongside an emoji reflecting the sentiment of the comment. Comments with inappropriate content are automatically filtered out, discarded, and logged in the backend.
+
+As the bookstore owner, you'll receive notifications via Slack each time if a comment containing “bad word” is submitted.
+
+## **Learning Goal**
+
+![Learning Goal Image](images/3.png)
+
+You will learn Event-Driven Architecture (EDA) and how it contrasts with traditional application designs that use microservices and REST APIs. You'll learn:
+
+- **The Fundamentals of EDA**: Explore the core principles of event-driven architecture and how it enhances responsiveness and scalability in applications.
+- **Comparative Insights**: Understand the differences between EDA and traditional architectures, highlighting the benefits and use cases of each.
+- **Practical Application**: Discover how to transition your existing applications to an event-driven model, utilizing the powerful features of Knative Eventing.
+
+![Knative Framework](images/4.png)
+
+Knative is a powerful framework that operates on top of Kubernetes. This tutorial will guide you through:
+
+- **Setting Up Your Cluster**: You'll start by spinning up your own Kubernetes cluster, which is the foundation for deploying and managing containers.
+- **Exploring Knative**: Gain hands-on experience with key Knative concepts and components.
+- **Some example use cases of Knative**.
+
+By the end of this tutorial, you will not only understand these concepts but also feel comfortable implementing them, empowering you to build robust, scalable event-driven applications with Knative.
+
+## **BookStore Architecture**
+
+![Bookstore Architecture](images/5.png)
+
+The bookstore application consists of a few components.
+
+### User Interface
+
+A frontend Next.js application that interacts with these services. It is a web page where users can select a book to view its details, ratings, and reviews.
+
+### Database Service
+
+An in-memory PostgreSQL instance on Kubernetes, storing all user comments.
+
+### Book Reviews Service
+
+A node.js web server that will perform the event forwarding, database operation, and handling the websocket connection.
+
+### Notification Service
+
+An Apache Camel-K pipe that connects our event-driven architecture with a third-party webhook: Slack. It receives the CloudEvent and sends it as a message to a Slack Workspace.
+
+### ML Models Service
+
+There are 2 Machine learning workflows that can conduct sentiment analysis on user’s review comment and hateful word sanity check. You will be using Knative Sequence to make sure they are executed in order.
```

Comment:
```suggestion
There are 2 Machine learning workflows that can conduct sentiment analysis on user’s review comment and hateful word sanity check. You will be using a [Knative Sequence](https://knative.dev/docs/eventing/flows/sequence/) to make sure they are executed in order.
```

Maybe add the link to what a Sequence is, in case users are new to Knative :shrug: